src/main/kotlin/io/zerobase/smarttracing/models/Models.kt

```diff
@@ -11,6 +11,7 @@ inline class ScanId(override val value: String): Id
 inline class OrganizationId(override val value: String): Id
 inline class UserId(override val value: String): Id
 inline class ScannableId(override val value: String): Id
+inline class CheckInId(override val value: String): Id
```

Comment:
We have been using ScanId in the section using legacy code

Reply:
Should I switch over?

Reply:
i already switched you over

Reply:
Thanks!